src/ActorsPanel.tsx

```diff
@@ -88,21 +89,25 @@ export const ActorsPanel: React.FC = () => {
             flexDirection="row"
             alignItems="center"
             padding={2}
-            borderBottom="1px solid"
-            borderBottomColor="gray.500"
             marginTop="0"
             opacity={currentSessionId === sessionId ? 1 : 0.5}
+            onClick={() => {
+              simActor.send({ type: 'SERVICE.FOCUS', sessionId });
+            }}
+            background={
+              currentSessionId === sessionId ? 'whiteAlpha.200' : 'transparent'
+            }
+            borderRadius="md"
+            _hover={{
+              background: 'whiteAlpha.100',
+            }}
+            marginBottom="2"
+            cursor="pointer"
```

Comment:
It IMHO would make slightly more sense to wrap the whole content of the `ListItem` in `Link` - if you are using the `Link`. That way you wouldn't have add `cursor: pointer` on this element and `onClick` would be placed on a slightly more appropriate (semantically) element